java/arcs/sdk/android/storage/service/StorageService.kt

```diff
@@ -242,5 +249,10 @@ class StorageService : ResurrectorService() {
                 action = storeOptions.actual.storageKey.toString()
                 putExtra(EXTRA_OPTIONS, storeOptions)
             }
+
+        fun createStorageManagerBindIntent(context: Context): Intent =
```

Comment:
As this is potentially a developer-facing API, KDoc would be very helpful here - explaining to the caller what will happen if they `bindService` with the resulting Intent from this.